spec/fixtures/custom_nginx.template

```diff
@@ -64,6 +64,10 @@ http {
         kong.init_worker()
     }
 
+    rewrite_by_lua_block {
+        kong.rewrite()
+    }
```

Comment:
Hmmmm, this is indeed an oversight (that the `rewrite_by_lua_block` is not included in this fixture template), but I think this should be placed at the same location it is in the default template? That would be, under `location /`, see: https://github.com/Mashape/kong/blob/master/kong/templates/nginx_kong.lua#L112-L114

No?